applications/Preprocessor/lib/MeshSource.h

```diff
@@ -159,8 +163,17 @@ class MeshSource2 {
          */
         std::string zoneName;
 
+        /**
+         * Dimension of the Element
+         */
+        size_t dimension;
```

Comment:
I think that it may be good to clarify that this is a courtesy for implementing a reader. That the a user of the source should not assume that it is set to any (sane) value.